### PR TITLE
Clp tut

### PR DESCRIPTION
**Pull request by utx0**
* markdown script for the clp video

### CODE REVIEW

File: x/clp/client/cli/tx.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ func GetTxCmd(cdc *codec.Codec) *cobra.Command {
 
 func GetCmdCreatePool(cdc *codec.Codec) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "create-pool",
+		Use:   "create-pool --from [key] --symbol [assert-symbol] --nativeAmount [amount] --externalAmount [amount]",
```
Comment (Jedi2002) [inline x/clp/client/cli/tx.go:41]: I think there's a typo here

Comment (utx0) [reply x/clp/client/cli/tx.go:41]: where??

Comment (intl-man) [reply x/clp/client/cli/tx.go:41]: @utx0 you have `assert-symbol` when it should be `asset-symbol`

**Review by intl-man — DISMISSED** [review]

**Review by intl-man — DISMISSED** [review]

File: docs/tutorials/clp tutorial.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+* Peggy ethBridge: https://github.com/Sifchain/sifnode/blob/develop/docs/tutorials/peggy%20tutorial.md
+
+#### Dependencies:
+
+    0. `git clone git@github.com:Sifchain/sifnode.git`
+        
+
+#### What are they
+
+Continues liquidity pools are a way to pool assets that can then be used in a decentralised blockchain to enable the exchange/swapping from one asset to another without the need for a private off chain exchange. At the sametime providing a yield/return to the liquidity providers based on the pool units each provider has within a pool.
```
Comment (banshee) [inline docs/tutorials/clp tutorial.md:18]: Continuous should be continuous

Comment (utx0) [reply docs/tutorials/clp tutorial.md]: fixed

**Review by intl-man — APPROVED** [review]

**Pull request merged**